crates/uv/src/commands/venv.rs

```diff
@@ -150,21 +150,32 @@ async fn venv_impl(
         Some(&reporter),
     )
     .await
-    .into_diagnostic()?
-    .into_interpreter();
+    .into_diagnostic()?;
+
+    let managed = python.source().is_managed();
```

Comment:
Alternatively we could check if this is a child of the managed Python version directory.